tools/wptrunner/requirements.txt

```diff
@@ -7,3 +7,4 @@ pillow==6.2.2  # pyup: <7.0
 urllib3[secure]==1.25.10
 requests==2.24.0
 six==1.15.0
+zstandard==0.14.0
```

Comment:
I'm nervous about this, because only the frontend requires it. Installing it here might break the gecko CI. We could have a seperate requirements file in `tools/wpt` just for things used by the frontend.

Reply:
You're right. This is only used by `manifest/download` via `wpt run` front-end. Let me drop this (along with the warning about coexistence of `install` and `requirements`).